app/src/components/dialog/ErrorDialog.tsx

```diff
@@ -93,21 +96,25 @@ export const ErrorDialog = (props: IErrorDialogProps) => {
       <DialogTitle id="alert-dialog-title">{props.dialogTitle}</DialogTitle>
 
       <DialogContent>
-        <DialogContentText id="alert-dialog-description">{props.dialogText}</DialogContentText>
-        {props.dialogError && <DialogContentText id="alert-dialog-description">{props.dialogError}</DialogContentText>}
+        <Stack gap={2}>
+          <DialogContentText id="alert-dialog-description">{props.dialogText}</DialogContentText>
+          {props.dialogError && (
+            <DialogContentText id="alert-dialog-description">{props.dialogError}</DialogContentText>
+          )}
 
-        {props?.dialogErrorDetails?.length ? (
-          <>
-            <Button color="primary" onClick={() => setIsExpanded(!isExpanded)}>
-              {isExpanded ? 'Hide detailed error message' : 'Show detailed error message'}
-            </Button>
-            <Collapse in={isExpanded}>
-              <ErrorDetailsList errors={props.dialogErrorDetails} />
-            </Collapse>
-          </>
-        ) : (
-          <></>
-        )}
+          {props?.dialogErrorDetails?.length ? (
+            <Box>
+              <Link color="primary" onClick={() => setIsExpanded(!isExpanded)}>
+                {isExpanded ? 'Hide detailed error message' : 'Show detailed error message'}
+              </Link>
+              <Collapse in={isExpanded}>
+                <ErrorDetailsList errors={props.dialogErrorDetails} />
+              </Collapse>
+            </Box>
+          ) : (
+            <></>
```

Comment:
Does this need to be a fragment for the stack? Or does null work?

Reply:
changed to `null`